### PR TITLE
Cli

### PR DESCRIPTION
**Pull request by MahanBi**
cli is written. All you have to do is write the shell script file.

### CODE REVIEW

File: panther/ui/cli.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+ui_folder = Path(__file__).resolve().parent
+OS_name = os.name
+
+
+# TODO: write shell scrip for making app and project
+def parser(args):
+    match args:
+        case {'version': True}:
+            print('0.1.4')
```
Comment (AliRn76) [inline panther/ui/cli.py:16]: return the version from package __init__

File: panther/main.py [hunk 2]
```diff
@@ -121,19 +122,12 @@ def collect_middlewares(self):
...
-            if _middleware[0].split('/')[0] == 'panther':
-                _middleware_name = _middleware[0].split('/')[-1]
-                if _middleware_name == 'db.py':
-                    from panther.middlewares.db import Middleware
-                elif _middleware_name == 'redis.py':
-                    from panther.middlewares.redis import Middleware
-                else:
-                    logger.error(f'{_middleware[0]} Does Not Found.')
-                    continue
+            if not _middleware == "":
```
Comment (AliRn76) [inline panther/main.py:125]: next time push these things to another branch

File: panther/ui/cli.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+def main():
+    arg_parser = ArgumentParser(description='Panther, Fast & Easy Python Framework.')
+    arg_parser.add_argument('-v', '--version', action='store_true', help='panther version.')
+    sub_parser = arg_parser.add_subparsers(title='Main Commands')
+    # make project command
+    make_project_parser = sub_parser.add_parser(name='makeproject', help='make project files.')
+    make_project_parser.add_argument('name', type=str, help='project name.')
+    make_project_parser.add_argument('-path', default=getcwd(), type=str, help='path')
+    # makeapp command
+    makeapp = sub_parser.add_parser('makeapp', help='make app files.')
```
Comment (AliRn76) [inline panther/ui/cli.py:38]: change makeapp to create

**Comment by AliRn76** [comment]
And lets talk about the way you created the directories :)

**Comment by MahanBi** [comment]
After receiving the values. Detect the operating system via os.name. Then the desired script file is executed. Windows uses xcopy for copying, which is available in Windows 8.1 and later.
The files are in the template folder for the app or project creation. A folder with the entered name is created, then it copies the files from the template to that folder.

**Comment by MahanBi** [comment]
change collect_middlewares to default

**Pull request merged**